charts/fluentd/values.yaml

```diff
@@ -3,7 +3,8 @@ fullnameOverride: ""
 
 # DaemonSet, Deployment or StatefulSet
 kind: "DaemonSet"
-
+# azureblob, cloudwatch, elasticsearch7, elasticsearch8, gcs, graylog , kafka, kafka2, kinesis, opensearch     
+outputType: elasticsearch7
```

Comment:
Change to `variant` per @stevehipwell request

Reply:
I will change it, i just put previously as _outputType_ because of the output config later in the values file to configure the output, although there's an option to change the output type config it was hardcoded to use only elasticsearch only fluentd image which posed issues for me when trying to implement it with opensearch.

i will push the changes, should i bump the versions on the helm chart ?